### ISSUE TITLE
programs.noisetorch.enable option doesn't add noisetorch package to environment.systemPackages

### ISSUE THREAD

**Issue by PAI5REECHO** [NixOS/nixpkgs#183069]
### Describe the bug
The [`programs.noisetorch.enable`](https://github.com/NixOS/nixpkgs/blob/5a0e0d73b944157328d54c4ded1cf2f0146a86a5/nixos/modules/programs/noisetorch.nix#L21-L27) option adds `noisetorch` to `$PATH` [via the security wrapper in `/run/wrappers/bin/`](https://github.com/NixOS/nixpkgs/blob/5a0e0d73b944157328d54c4ded1cf2f0146a86a5/nixos/modules/programs/noisetorch.nix#L21-L27) but doesn't add `noisetorch` to `environment.systemPackages`. It's pretty common for `programs.*.enable` options to add packages to `environment.systemPackages` (see [`programs.htop.enable`](https://github.com/NixOS/nixpkgs/blob/5a0e0d73b944157328d54c4ded1cf2f0146a86a5/nixos/modules/programs/htop.nix#L48-L50) & [`programs.git.enable`](https://github.com/NixOS/nixpkgs/blob/5a0e0d73b944157328d54c4ded1cf2f0146a86a5/nixos/modules/programs/git.nix#L50) for example) in order to make the .desktop entries available to application launchers etc.

### Steps To Reproduce
1. Enable `programs.noisetorch.enable`
2. The `noisetorch` GUI is now available to be launched from a terminal but no .desktop entry is available to launch the UI from an application launcher because it's not in `environment.systemPackages`

### Expected behavior
`noisetorch` should be added to `environment.systemPackages`

### Notify maintainers
@PanAeon @legendofmiracles

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.4, NixOS, 22.11 (Raccoon), 22.11.20220619.0d68d7c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.0pre20220620_16c6c6c`
 - nixpkgs: `/nix/store/1njdlszpc3bchdcwk45c3ndb0nfmwqcr-source`
```

**Comment by Shawn8901** [comment]
I completely agree with you on that. #184027 should add it to `environment.systemPackages`, sadly the package does not contain a `.desktop` neither but that should be fixed with #184022

**Comment by onny** [comment]
As mentioned in the PR, isn't

**Comment by Shawn8901** [comment]
like written on the PR if you could point me to any solution not doing it that way i would be very appreciative. When i look thru other occurences for `security.wrappers` in the repository i can just find either having it "missing" to add to  `environment.systemPackages` or adding it the same way the PR did. 
If there is any other way, i would be happy to apply that, but cant find any.
